server/stream.go

```diff
@@ -576,6 +582,16 @@ func (a *Account) addStreamWithAssignment(config *StreamConfig, fsConfig *FileSt
 	c := s.createInternalJetStreamClient()
 	ic := s.createInternalJetStreamClient()
 
+	// Work out the stream ingest limits.
+	mlen := s.opts.JetStreamMaxBufferMsgs
+	msz := uint64(s.opts.JetStreamMaxBufferSize)
+	if mlen == 0 {
+		mlen = streamDefaultMaxQueueMsgs
+	}
+	if msz == 0 {
+		mlen = streamDefaultMaxQueueBytes
```

Comment:
should be this maybe?
```suggestion
		msz = streamDefaultMaxQueueBytes
```

Reply:
Good catch, fixed.

Reply:
Different test needed maybe to cover `JetStreamMaxBufferSize` taking effect?

Reply:
Done.